api/src/main/java/io/opentelemetry/trace/attributes/SemanticAttributes.java

```diff
@@ -16,6 +16,9 @@
 
 package io.opentelemetry.trace.attributes;
 
+import static io.opentelemetry.trace.attributes.LongAttributeSetter.longKey;
```

Comment:
Attributes are used in other places like Resource would be good to have this helper in common.